src/st/dai/e_init.c

```diff
@@ -153,12 +153,16 @@ EInit OVL_EXPORT(EInitBreakable) = {ANIMSET_DRA(3), 0, 0, PAL_DRA(0), 0};
 
 #include "../e_init_common.h"
 
-// Looks to be unused
+// Unused, not in psp
+#ifndef VERSION_PSP
```

Comment:
Are these ifndefs required or does PSP automatically deadstrip these?

Reply:
There is a bunch of these left around the place from before Xee got deadstripping working which can likely be removed